benchmark/napi/napi_property_benchmark/binding.gyp

```diff
@@ -0,0 +1,14 @@
+{
+  "targets": [
+    {
+      "target_name": "napi_property_benchmark",
+      "sources": [
+        "binding.c"
+      ],
+      "cflags": ["-std=c++11"],
+      "include_dirs": [
+        "<!(node -e \"require('nan')\")"
```

Comment:
This is node-api addon and should not depend on `nan`. I think the benchmark could be a follow up since the APIs added are experimental.

I created https://github.com/nodejs/node/pull/54012 for existing `node_api_create_property_key_utf16`.